website/docs/docs/cloud/secure/redshift-privatelink.md

```diff
@@ -79,7 +79,7 @@ Creating an Interface VPC PrivateLink connection requires creating multiple AWS
 
 On the provisioned VPC endpoint service, click the **Allow principals** tab. Click **Allow principals** to grant access. Enter the ARN of the root user in the appropriate production AWS account and save your changes.
 
- - Principal: `arn:aws:iam::346425330055:root`
+ - Principal: `arn:aws:iam::346425330055:role/MTPL_Admin` (_or_ `arn:aws:iam::346425330055:root`)
```

Comment:
I just removed the root principal. That's what everyone used to use until we added the role, which is a more secure practice. I left it in there in case someone was still using root and wondered why theirs was different, but I'm not super worried about it. This is more clear and precise.